deployment/build.gradle

```diff
@@ -12,6 +12,23 @@ dependencies {
     runtime("javax.xml.bind:jaxb-api:$jaxbVersion")
 }
 
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.ajoberstar:grgit:1.1.0'
```

Comment:
Can you add this to the other plugin dependencies (above) like so:

`id "org.ajoberstar.grgit" version "2.3.0"`

(and 2.3.0 is a more recent version, fyi)

Reply:
i.e. in the main `./build.gradle` script at the top of the file.